packages/grid/_modules_/grid/context/GridPropsContext.ts

```diff
@@ -0,0 +1,4 @@
+import * as React from 'react';
+import { GridComponentProps } from '../GridComponentProps';
+
+export const GridPropsContext = React.createContext<GridComponentProps | undefined>(undefined);
```

Comment:
React doesn't give any tip in the dev tools. Prefer this pattern:

https://github.com/mui-org/material-ui/blob/4300a41e99f53704ebcd09bcb3c06724ac014171/packages/material-ui/src/Table/TableContext.js#L8-L10